src/components/LabeledSlider.tsx

```diff
@@ -161,59 +203,86 @@ export default function LabeledRangeSlider(inputProps: LabeledRangeSliderProps):
+
+  // Change value/onChange properties based on the type of slider
+  const sliderProps: SliderSingleProps | SliderRangeProps =
+    props.type === "value"
+      ? {
+          value: props.value,
+          onChange: handleValueChange,
+        }
+      : {
+          value: [props.min, props.max],
+          onChange: (value: [number, number]) => handleRangeChange(value[0], value[1]),
+          range: { draggableTrack: true },
+        };
 
   return (
     <ComponentContainer>
-      <InputNumber
-        ref={minInput}
-        size="small"
-        style={{ width: "80px" }}
-        value={props.min}
-        onPressEnter={handleMinInputChange}
-        onBlur={handleMinInputChange}
-        controls={false}
-        disabled={props.disabled}
-      />
+      {
+        // Conditionally render either min or value input
+        props.type === "value" ? (
+          <InputNumber
+            ref={valueInput}
+            size="small"
+            style={{ width: "80px" }}
+            value={props.value}
+            onPressEnter={handleValueInputChange}
+            onBlur={handleValueInputChange}
+            controls={false}
+            disabled={props.disabled}
+          />
+        ) : (
+          <InputNumber
+            ref={minInput}
+            size="small"
+            style={{ width: "80px" }}
+            value={props.min}
+            onPressEnter={handleMinInputChange}
+            onBlur={handleMinInputChange}
+            controls={false}
+            disabled={props.disabled}
+          />
+        )
+      }
       <SliderContainer>
         <Slider
+          {...sliderProps}
           min={props.minSliderBound}
           max={props.maxSliderBound}
-          range={{ draggableTrack: true }}
-          value={[props.min, props.max]}
           disabled={props.disabled}
-          onChange={(value: [number, number]) => {
-            handleValueChange(value[0], value[1]);
-          }}
           marks={marks}
           step={stepSize}
           // Show formatted decimals in tooltip
           // TODO: Is this better than showing the precise value?
           tooltip={{
-            formatter: (value) => numberToStringDecimal(value, props.maxDecimalsToDisplay),
+            formatter: numberFormatter,
             open: props.disabled ? false : undefined, // Hide tooltip when disabled
           }}
         />
+
         <SliderLabel $disabled={props.disabled}>{minSliderLabel}</SliderLabel>
         <SliderLabel $disabled={props.disabled}>{maxSliderLabel}</SliderLabel>
       </SliderContainer>
-      <InputNumber
-        ref={maxInput}
-        size="small"
-        type="number"
-        style={{ width: "80px" }}
-        value={props.max}
-        onPressEnter={handleMaxInputChange}
-        onBlur={handleMaxInputChange}
-        controls={false}
-        disabled={props.disabled}
-      />
+      {props.type === "range" ? (
```

Comment:
nit: 
```suggestion
      {props.type === "range" && (
```
and then remove the null